### ISSUE TITLE
Tensor name "InceptionV3/AuxLogits/Conv2d_2a_1x1/BatchNorm/beta" not found in checkpoint files

### ISSUE THREAD

**Issue by IsCoelacanth** [dongyp13/Targeted-Adversarial-Attack#3]
We cloned your repo and we're trying to replicate the results but we keep facing this error. skipping this checkpoint doesn't doesnt resolve it. Similar errors come up in other model too. Could you help us in debugging it.

system: Python 2.7 and 3.6, tensorflow CPU and GPU error persists between systems.

**Comment by senlin-ali** [comment]
hi,i have the same quesion ,did you solve it?
